ignition/what-is-ignition.md

```diff
@@ -27,8 +25,6 @@ octal or decimal. Ignition's configuration is also versioned which allows it to
 be improved in the future without having to worry as much about maintaining
 backward compatibility.
 
-[cloudinit]: https://github.com/coreos/coreos-cloudinit
-
 ### Ignition Only Runs Once ##
```

Comment:
Can you fix up the trailing `#` while you are in here? The guidelines specify that these are no longer included.